src/tests/ftest/recovery/pass_2.yaml

```diff
@@ -0,0 +1,23 @@
+hosts:
+  test_servers: 2
+
+timeout: 360
+
+server_config:
+  engines_per_host: 2
+  engines:
+    0:
+      storage: auto
+      fabric_iface_port: 31416
+      log_file: daos_server_0.log
+      pinned_numa_node: 0
+      fabric_iface: ib0
+    1:
+      storage: auto
+      fabric_iface_port: 31516
+      log_file: daos_server_1.log
+      pinned_numa_node: 1
+      fabric_iface: ib1
+
+pool:
+  size: 100000000000 # 100G
```

Comment:
Does `100G` not work as expected?

Reply:
Yes. I was doing some arithmetic before and forgot to bring it back to human readable format.